docs/release-notes.md

```diff
@@ -1,230 +1,123 @@
+- see change log for full list of changes
+#### RPC-Go
+- **build:** adds support for c-shared buildmode
+- **logging:** adds support for fine grained control of log output
+- **heci:** adds retry for device busy
+- **lms:** rewrite lme communication in go
+- **main:** eliminate need for CGO if not building library
+- see change log for full list of changes
+#### MPS Router
+- **env:** add option to override default mps host (#0b5fdd9) 
+- **healthcheck:** adds flag for checking db status (#b3360c7) 
+- see change log for full list of changes
+#### Sample Web UI
+- cira + static ip configuration (#9c0f06a) 
+- Show DHCP/Static on Profiles page (#22432c2) 
+- **redirection:** adds property to track redirection state (#aaaaff8) 
+- fix status code expectation (#2f080e9) 
+- see change log for full list of changes
+#### UI Toolkit
+- aligns cursor properly when scrolling (#f4d4669) 
+- **kvm:** scrolling offset for mouse is fixed in ui-toolkit 2.0.6 (#dabc394)
+- UI Toolkit - Angular *** Breaking Change ***  components now require angular 13
+- see change log for full list of changes
+#### WSMAN-MESSAGES
+- **createBody:** handles LinkPolicy arrays (#96ba28e) 
+- **amt:** remove un-implemented method (#07bf552)
+- see change log for full list of changes
+- **cira:** adds MpsType set to both (#5fb6763)
+- **AMT:** Adds Remote Access Policy Applies to MPS (#b8ca4b9) 
 
 ## Resolved Issues
 #### RPS
-- **[AMT Wi-Fi Configuration not supported on non-Windows systems](https://github.com/open-amt-cloud-toolkit/rps/issues/349):** Known Issue
-- **[Remove mpsRootCertificate parameter from createCiraConfig route](https://github.com/open-amt-cloud-toolkit/rps/issues/461):** Question
+- **[Vault health check can return a false positive result](https://github.com/open-amt-cloud-toolkit/rps/issues/629):** Bug
 #### MPS
-- **[req.query.$status should be req.query.status instead in src/routes/devices/getAll.ts](https://github.com/open-amt-cloud-toolkit/mps/issues/508):** Bug
-- **[Audit Log calls never respond on specific versions of AMT](https://github.com/open-amt-cloud-toolkit/mps/issues/301):** Known Issue
+- **[Allow filtering devices by hostname](https://github.com/open-amt-cloud-toolkit/mps/issues/548):** Enhancement
 #### RPC-Go
-- **[json output](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/22):** Enhancement
-- **[Invalid comment lines on Dockerfile](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/18):** Bug
-
+- **[Root certificates not present in AMT device](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/42):** Bug
+- **[both c rpc and rpc-go seem to be missing the DNS Suffix set in MEBx](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/26):** Needs More Investigation
+- **[rpc-go missing some output compared to the c version, and intermittently misses other pieces](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/25):** Bug
```

Comment:
'rpc-go missing some....' is listed under both Closed Issues and Open Issues.  The Issue itself is closed, is this officially closed?

Reply:
Fixed